src/main/groovy/projects/dktk/v2/ExportResourceMappingConfig.json

```diff
@@ -110,6 +110,11 @@
       "selectFromCxxEntity": "DIAGNOSIS",
       "transformByTemplate": "todUrsache",
       "exportToFhirResource": "Observation"
+    },
```

Comment:
This file is not up to date anymore, please merge the current version from master first.